.vib/etcd/goss/vars.yaml

```diff
@@ -0,0 +1,16 @@
+binaries:
+  - yq
+  - etcd
+  - etcdctl
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/etcd/data
+      - /opt/bitnami/etcd/certs
+  - mode: "0755"
```

Comment:
I would not implicitly check for permissions here, as this is not set in `postunpack` for this folder.